gatsby-node.esm.js

```diff
@@ -109,10 +109,12 @@ export const sourceNodes = async ({
         );
     });
     // This must be done after so all author bios exist
-    snootyArticles = snootyArticles.map(
-        ({ slug, doc }) =>
-            new SnootyArticle(slug, doc, slugContentMapping, pathPrefix)
-    );
+    if (!Boolean(process.env.GATSBY_PREVIEW_MODE)) {
```

Comment:
Any particular reason we use `Boolean()` vs simply the ! operator? If undefined then this would still run if we had `!process.env.GATSBY_PREVIEW_MODE`

Or is the env coming in as a string?

Reply:
Boolean(undefined) or Boolean(false) is false. This is just saying Preview mode is false by default, meaning we did not provide preview mode env variable some where, it evaluates false by default.

Reply:
Just my way, of evaluating. Can also use !process.env.GATSBY_PREVIEW_MODE.

Reply:
I'm indifferent so this is fine here!